tests/testing-plugins/testing_policy.c

```diff
@@ -243,8 +243,6 @@ generateNonce_testing(const UA_SecurityPolicy *securityPolicy,
                       UA_ByteString *out) {
     ck_assert(securityPolicy != NULL);
     ck_assert(out != NULL);
-    if(out->length != 0)
```

Comment:
Why remove this check?

Reply:
You're right. It could be left in.

The nonce is generated with UA_ByteString_alloc.
If that invariant fails, we have loads of other problems as well.
It should be detected basically everywhere.